cutten-web/src/components/Main.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import {
+  gql
+} from 'apollo-boost';
+
+import {
+  Query
+} from 'react-apollo';
+
+import {
+  ThemeProvider,
+  theme,
+  Notice
+} from 'kingsbury/lib';
+
+import SelectButton from './SelectButton'
+import UploadButtonV2 from './UploadButtonV2'
+import AnalyzeButton from './AnalyzeButton'
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      file: null,
+      isUploaded: null
+    };
+  }
+
+  onFileSelect = ({ target: { files } }: React.ChangeEvent<HTMLInputElement>) => {    
+
+    if (!files) {
+      return;
+    }
+
+    if (files.length === 1) {
+      this.setState({
+        file: files[0]
+      });
+    }
+  }
+
+  render() {
+    let { file } = this.state;
+
+    return (
```

Comment:
We just want to show 3 buttons in a row? what happens when I click Upload or Analyze when there is no file?

Reply:
You can't click them, they're disabled